db/document.go

```diff
@@ -1192,6 +1193,14 @@ func (doc *Document) UnmarshalWithXattr(ctx context.Context, data []byte, xdata
 			Cas: casOnlyMeta.Cas,
 		}
 		doc._rawBody = data
+	case DocUnmarshalVV:
+		tmpData := SyncData{}
+		unmarshalErr := base.JSONUnmarshal(xdata, &tmpData)
+		if unmarshalErr != nil {
+			return pkgerrors.WithStack(base.RedactErrorf("Failed to UnmarshalWithXattr() doc with id: %s (DocUnmarshalVV).  Error: %v", base.UD(doc.ID), unmarshalErr))
```

Comment:
`pkgerrors` is kind of old now. We want to avoid using it in new code and instead use `%w` to wrap our Go errors with the stdlib functions so we can use `errors.Is`/`errors.As`